src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -310,12 +310,24 @@ function getFilesRecursively(dir: string, isDependenciesFolder: boolean): string
     return files
 }
 
-function getProjectDependencies(modulePath: string): string[] {
+function getProjectDependencies(buildCommand: CodeTransformMavenBuildCommand, modulePath: string): string[] {
     // Make temp directory
     const folderName = `${CodeWhispererConstants.dependencyFolderName}${Date.now()}`
     const folderPath = path.join(os.tmpdir(), folderName)
 
-    const baseCommand = 'mvn'
+    let baseCommand = buildCommand as string
+    if (baseCommand === 'mvnw') {
```

Comment:
Question: It looks like IntelliJ we run `./mvnw` first and `mvn` second. I think we should align on the order. See IntelliJ Implementation in [CodeModernizerSessionContext.kt](https://github.com/aws/aws-toolkit-jetbrains/blob/main/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt#L190)

Reply:
Agreed & fixed